portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -595,4 +1137,184 @@ definition {
 		}
 	}
 
+	@description = "LPS-192149. Verify if the Headless Action Key is not mandatory and contains an tooltip Icon with an message"
+	@priority = 3
+	test HeadlessActionKeyIsNotMandatory {
+		task ("And clicks on Create Item Action button") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then the Headless Action Key field is not mandatory") {
+			AssertElementNotPresent(
+				key_fieldName = "Headless Action Key",
+				locator1 = "DataSet#MANDATORY_FIELD");
+		}
```

Comment:
To check a field is not mandatory you have to try to save it and check that you can save it. Checking the mandatory icon is not enough. Please add this verification to the test.